docs/packs_format.md

```diff
@@ -0,0 +1,59 @@
+---
+id: packs_format
```

Comment:
Change the id to packs-format and the file name. Let's keep things consistent. Also need to add the id to sidebars.